test/edu-benefits/1995/config/contactInformation.unit.spec.jsx

```diff
@@ -0,0 +1,130 @@
+import React from 'react';
+import { findDOMNode } from 'react-dom';
+import { expect } from 'chai';
+import ReactTestUtils from 'react-addons-test-utils';
+import Form from 'react-jsonschema-form';
+
+import { DefinitionTester } from '../../../util/schemaform-utils.jsx';
+import formConfig from '../../../../src/js/edu-benefits/1995/config/form';
+
+describe('Edu 1995 contactInformation', () => {
+  const { schema, uiSchema } = formConfig.chapters.personalInformation.pages.contactInformation;
+  it('should render', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+
+    const inputs = ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'input').concat(
+      ReactTestUtils.scryRenderedDOMComponentsWithTag(form, 'select')
+    );
+
+    expect(inputs.filter(input => input.id.startsWith('root_preferredContactMethod')).length)
+      .to.equal(3);
+    expect(inputs.filter(input => input.id.startsWith('root_veteranAddress')).length)
+      .to.equal(6);
+    expect(inputs.filter(input => input.id.startsWith('root_view:otherContactInfo')).length)
+      .to.equal(4);
+  });
+  it('should render required fields', () => {
+    const form = ReactTestUtils.renderIntoDocument(
+      <DefinitionTester
+          schema={schema}
+          data={{}}
+          uiSchema={uiSchema}/>
+    );
+
+    const formDOM = findDOMNode(form);
+    ReactTestUtils.findRenderedComponentWithType(form, Form).onSubmit({
+      preventDefault: f => f
+    });
+
+    expect(formDOM.querySelectorAll('.usa-input-error').length).to.equal(6);
+  });
+  it('should not render confirm email on review page', () => {
```

Comment:
That's a good suggestion. A lot of these pages won't have have review specific behavior to test, for what it's worth.